docs/UserGuide.md

```diff
@@ -77,13 +77,19 @@ Examples:
 * `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01`
 * `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 t/criminal`
 
-### Listing all assignments : `list`
+### Listing all assignments and lessons : `list NUMBER`
 
-Shows a list of all assignments in your schedule.
+- Shows a list of all assignments and lessons in your schedule within `NUMBER` 
+number of weeks.
+- A week in this context starts with Monday and ends with Sunday.
 
-Format: `list`
+Format: `list NUMBER`
 
-Example: `list`
+Examples: 
+- `list 2` displays all your assignments and lessons within 
+the current and the next week.
```

Comment:
Maybe changing it to "displays all your assignments and lessons for the next 2 weeks" would be more succinct?

Reply:
Next two weeks might sound like this week is excluded

Reply:
How about "displays all your lessons and assignments for the next 2 weeks (including the current week)"?

Reply:
next two weeks is already excluding this week

Reply:
How about "displays all your lessons and assignments for 2 weeks (including the current week)"?

Reply:
@printinghelloworld sounds good :)

Reply:
Amended